python/Ganga/new_tests/GPI/TestJob.py

```diff
@@ -100,4 +103,12 @@ def testJobCopy(self):
         self.assertEqual( j2.postprocessors[0].searchStrings, ["my search"] )
         self.assertTrue( isType(j2.splitter, GenericSplitter) )
         self.assertEqual( j2.splitter.attribute, "application.args" )
-        self.assertEqual( j2.splitter.values, ['arg 1', 'arg 2', 'arg 3'])
\ No newline at end of file
+        self.assertEqual( j2.splitter.values, ['arg 1', 'arg 2', 'arg 3'])
+
+    def test_job_equality(self):
+        from Ganga.GPI import Job
+        j = Job()
+        j2 = j.copy()
+        j3 = Job(j)
+        assert j == j2
+        assert j2 == j3
```

Comment:
Can you check the behavior of un-proxied copies here or in another short test as well